tests/pfcwd/test_pfc_config.py

```diff
@@ -158,11 +158,15 @@ def stop_pfcwd(duthosts, enum_rand_one_per_hwsku_frontend_hostname):
     Returns:
         None
     """
-    yield
     duthost = duthosts[enum_rand_one_per_hwsku_frontend_hostname]
     logger.info("--- Stop Pfcwd --")
     duthost.command("pfcwd stop")
 
+    yield
```

Comment:
@auspham does it revert the change in https://github.com/sonic-net/sonic-mgmt/pull/5392/files#diff-2b591708001ec52b172b72036e80a4c635a8cc8476747d95d6fa8f59b0ca3981?

@lipxu  @antonptashnik could you please review this change?

Reply:
@ZhaohuiS yes, the problem with putting `yield` first is that it doesn't reset pfcwd to the default setting as we should be doing after this module